docs/examples/all/db.yaml

```diff
@@ -1,32 +1,33 @@
 name: database
-containers:
-- image: mariadb:10
-  env:
-  - name: MYSQL_ROOT_PASSWORD
-    value: rootpasswd
-  - name: MYSQL_PASSWORD
-    value: wordpress
-  - name: MYSQL_USER
-    value: wordpress
-  envFrom:
-  - configMapRef:
-      name: database
-  volumeMounts:
-  - name: database
-    mountPath: /var/lib/mysql
-  livenessProbe:
-    exec:
-      command:
-      - mysqladmin
-      - ping
-    initialDelaySeconds: 30
-    timeoutSeconds: 5
-  readinessProbe:
-    exec:
-      command:
-      - mysqladmin
-      - ping
-    initialDelaySeconds: 5
+deployments:
+- containers:
```

Comment:
I'm just going through all these examples, but does `- containers` need to be an array? Can't it just be `containers:` ?@kadel

Reply:
Emphasized this here: https://github.com/kedgeproject/kedge/issues/541

Reply:
Deployemnts should be an array like all the other (`services`, `configMaps` ....). It should be possible to define multiple deployments, and